monkeys/apple-tree/run.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+set -o pipefail
+
+
+. $(dirname ${BASH_SOURCE})/helpers.bash
+
+init_monkey
+
+
+API_TOKEN_PATH=${API_TOKEN_PATH:-/var/run/secrets/kubernetes.io/serviceaccount/token}
+KUBE_TOKEN=$(< ${API_TOKEN_PATH})
+KUBERNETES_PORT_443_TCP_ADDR=${KUBERNETES_PORT_443_TCP_ADDR:-kubernetes}
+
+ERROR_MSG="oh noez! the apple fell far from the tree; (pod: $POD_NAME) failed"
+
+if [ ${DEBUG} ]; then
+  set -x
+fi
+
+[ -z "$MAXTIMEOUT" ] && {
+  MAXTIMEOUT="10"
+}
+
+[ -z "$CONNECTTIMEOUT" ] && {
+  CONNECTTIMEOUT=3
+}
+
+[ -z "$SLEEP" ] && {
+  SLEEP=10
+}
+
+
+endpoint_debug_enable
+
+function run_client() {
+  while :
+  do
+    start_monitor
+    start_tcpdump
+    log_line "trying to access 1.1.1.1"
```

Comment:
Would be nice to factor this out into a `URL` variable like the other tests tend to. Not a biggie though.

Reply:
Done.